apps/etterlatte-behandling/src/main/kotlin/oppgave/OppgaveService.kt

```diff
@@ -182,6 +182,27 @@ class OppgaveService(
         }
     }
 
+    fun settPaaVent(
+        oppgaveId: UUID,
+        merknad: String,
+        status: Status,
+    ) {
+        val nyStatus = if (status == Status.PAA_VENT) Status.UNDER_BEHANDLING else Status.PAA_VENT
```

Comment:
Denne kjens ikkje heilt intuitiv. Trur eg synes det er tydelegare å ha ta av vent som ein separat ting, og heller gjera ingenting (eller eventuelt kaste exception) viss vi kjem hit for ei oppgåve som allereie står på vent.

Reply:
Men det kan veldig fint vente til neste iterasjon i så fall.

Reply:
ok, jeg må uansett utbedre noen småting, så jeg kan sette det på listen, sikkert bedre for dere å få ut noe nå